pages/privacy.tsx

```diff
@@ -0,0 +1,167 @@
+import { Main } from 'layouts/main'
+import { StyledList, Text } from 'components/global/text'
+import { GetServerSideProps } from 'next'
+import { getCommonServerSideProps } from 'components/utils/getCommonServerSideProps'
+import { SafeLink } from 'components/global/safeLink'
+
+type PrivacyProps = {
+  conferenceName: 'string'
+}
+
+export default function Privacy({ conferenceName }: PrivacyProps) {
+  return (
+    <Main title="Privacy Policy" description={`${conferenceName} privacy policy`}>
+      <Text tag="h1">Privacy Policy</Text>
+
+      <Text>
+        <strong>DDD WA Inc.</strong> is committed to providing quality services to you and this policy outlines our
+        ongoing obligations to you in respect of how we manage your Personal Information. We have adopted the Australian
```

Comment:
Missing space between `Principles(APPs)`?

Reply:
Rebecca provided new (and better formatted) content so have update the page.